.gitignore

```diff
@@ -1,3 +1,5 @@
 *~
 cloudbuild.yaml
 cloudbuild_test.yaml
+mkdebootstrap/mkimage.sh
```

Comment:
i'm not totally sure actually....@sharifelgamal can you comment? they're not committed to the repository currently, and just sit around as build artifacts after each build. not a huge deal, but prevent us from doing things like `git add -A` :)